standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestListPartitions.java

```diff
@@ -409,10 +409,13 @@ public void testListPartitionsByValues() throws Exception {
     assertEquals(0, partitions.size());
   }
 
-  @Test(expected = MetaException.class)
```

Comment:
Empty part_vals should not throw `MetaException` because we support it in `ObjectStore` and regard it as fetching all partitions:
https://github.com/apache/hive/blob/2d855b27d31db6476f18870651db6987816bb5e3/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java#L3890-L3893